src/microbe_stage/SpawnSystem.cs

```diff
@@ -90,9 +90,9 @@ public SpawnSystem(Node root)
     ///   Adds an externally spawned entity to be despawned
     /// </summary>
     public static void AddEntityToTrack(ISpawned entity,
-        float radius = Constants.MICROBE_SPAWN_RADIUS)
+        float radius = Constants.MICROBE_SPAWN_RADIUS + Constants.DESPAWN_RADIUS_OFFSET)
     {
-        entity.DespawnRadiusSquared = (int)(radius * radius) + Constants.DESPAWN_RADIUS_OFFSET_SQUARED;
+        entity.DespawnRadiusSquared = (int)Mathf.Pow(radius, 2.0f);
```

Comment:
Someone may have commented on this already, but I'd vastly prefer to keep the "radius * radius" code because I fear how inefficient calling Mathf.Pow here would be, because it converts all floats to doubles and might not have an optimized path for when the exponent is an integer.

Reply:
Gotcha, I'll do that to this and the other places.